docs/_functions/domain/IGNORE.md

```diff
@@ -1,40 +0,0 @@
----
-name: IGNORE
----
-
```

Comment:
Rather than deleting this file, how about just a brief note that it has been renamed to be IGNORE_NAME().  IGNORE() still works but has been deprecated, please use the new name, etc. etc.